source/features/bypass-checks.tsx

```diff
@@ -15,9 +15,16 @@ async function bypass(detailsLink: HTMLAnchorElement): Promise<void> {
 	}
 
 	const {details_url: detailsUrl} = await api.v3(`check-runs/${runId}`);
```

Comment:
Have they implemented a v4 in the meanwhile?

Reply:
Yes, but we have to fetch the runs through each commit, it's quite cumbersome:

```graphql
{
  repository(name: "refined-github", owner: "refined-github") {
    object(expression: "869d9a7d818e029828a0ff2db0b6d3d4b389c44a") {
      ... on Commit {
        checkSuites(first: 10) {
          nodes {
            checkRuns(first: 10) {
              nodes {
                detailsUrl
              }
            }
          }
        }
      }
    }
  }
}
```